contents/tutorials/performance-metrics.md

```diff
@@ -8,93 +8,126 @@ featuredImage: todo
+- For a site whose main goal is to attract SEO traffic and is looking to improve their [PageSpeed Insights performance score](https://pagespeed.web.dev/), then Page Loaded will be the most important since this contributes the most to [Total Blocking Time](https://web.dev/tbt/), which has the [highest weighting](https://developer.chrome.com/docs/lighthouse/performance/performance-scoring/?utm_source=lighthouse&utm_medium=lr#weightings) when calculating the performance score.
+- For an e-commerce site with a checkout form with many input fields and complex validation, a high DOM Interactive time means that the form is slow to respond to user input. 
+- For a personal finance app, DOM Interactive ensures that users can quickly navigate and interact with various financial tools on offer. However, Page Loaded guarantees that all the data visualizations, such as charts and graphs, are fully loaded and functional. 
 
 ## Optimization cheat sheet
-If you're looking to improve any of the above metrics, here is a handy cheat sheet for where to look for opportunities. Each value in the table indicates how likely the metrics are to be affeted.
+If you're looking to improve any of the above metrics, here is a handy cheat sheet for where to look for opportunities. Each value in the table indicates how likely the metrics are to be affected.
 
-| What?   | First Contetful Paint  | DOM Interactive | Page Loaded |
+| What?   | First Contentful Paint  | DOM Interactive | Page Loaded |
 |--------|--------|--------|--------|
 | Page Size   | 🟢 High  | 🟢 High   | 🟢 High |
 | Server Response Time   | 🟢 High  | 🟢 High | 🟢 High |
 | Network Latency   | 🟢 High  | 🟢 High | 🟢 High |
-| Downloading Javacript  | 🟡 Medium (only if it blocks rendering) | 🟢 High | 🟢 High |
-| Executing Javacript  | 🟡 Medium (only if it blocks rendering) | 🟢 High | 🟢 High |
+| Downloading JavaScript  | 🟡 Medium (only if it blocks rendering) | 🟢 High | 🟢 High |
+| Executing JavaScript  | 🟡 Medium (only if it blocks rendering) | 🟢 High | 🟢 High |
+|API Requests |	🟡 Medium (only if they block rendering or are critical for initial content) |	🟡 Medium (only if they are for rendering critical content) |	🟢 High |
 | CSS Files: Downloading and Parsing  | 🟢 High | 🟡 Medium | 🟡 Medium |
-| Images: Downlading and Rendering| 🟡 Medium (only if they are part of the initial content) | 🔴 Low | 🟢 High |
+| Images: Downloading and Rendering| 🟡 Medium (only if they are part of the initial content) | 🔴 Low | 🟢 High |
 | Fonts: Downloading and Rendering  | 🟡 Medium (only if they are part of the initial content)  | 🔴 Low |  🔴 Low  |
 
-## Identifying Opportunities on Your Own Site
-Posthog's performance monitoring tool shows you the size and speed of different requests made during page load. You can also filter these by request type, such as API, JS, image, and CSS requets.
+## Identifying opportunities on your own site
```

Comment:
I'm a bit unsure on the strength of this section. I was struggling to find good examples to use here. Perhaps someone who is more familiar with the tool can add some insight as to what to include?